README.md

```diff
@@ -6,6 +6,14 @@ This is a mixin for Porter that provides the Docker Compose (docker-compose) CLI
 
 <img src="https://porter.sh/images/mixins/docker-compose.png" align="right" width="150px"/>
 
+## Install via Porter
+
+This will install the latest mixin release via the Porter CLI.
+
+```
+porter mixin install docker-compose --feed-url https://cdn.porter.sh/mixins/atom.xml
```

Comment:
that feed url is the default, so it should work with just `porter mixin install docker-compose`, want to try that out?

Reply:
Agree with you. On my side, to install the mixin, I just need to use the command: `porter mixin install docker-compose`.